plugins/woocommerce-admin/client/products/fields/variations/variations.scss

```diff
@@ -41,6 +45,7 @@
 		margin-left: -1px;
 		margin-right: -1px;
 		margin-bottom: -1px;
+		height: 84px;
```

Comment:
I think this should be a min-height in case the content is larger (such as having more tags).
<img width="551" alt="Screen Shot 2022-12-21 at 9 01 17 AM" src="https://user-images.githubusercontent.com/10561050/208962370-79005ad1-2766-4e7c-81c7-89d27314808c.png">



```suggestion
		min-height: 84px;
```